bin/plot_tree.py

```diff
@@ -104,11 +105,13 @@ def leaf_style(node, seqmeta, tp_colors, highlight_node=None):
             nstyle['fgcolor'] = tp_colors[seqmeta['timepoint']]
         nstyle['size'] = 14
         node.set_style(nstyle)
-    timepoints = filter(lambda x: x, seqmeta.get('cluster_timepoints', seqmeta['timepoints']).split(':'))
-    duplicities = [int(n) for n in seqmeta.get('cluster_timepoint_multiplicities', seqmeta['timepoint_multiplicities']).split(':') if n]
+    timepoints = seqmeta.get('cluster_timepoints', seqmeta['timepoints']).split(':')
+    duplicities = [int(n) for n in seqmeta.get('cluster_timepoint_multiplicities', seqmeta['timepoint_multiplicities']).split(':')]
```

Comment:
If we want to stay safe here we can use something like `(int(n) if n else 0) for n in ...`; Seems our approach to the fix here is to make sure that this is fixed upstream, but there was a reason this was biting me before, so may be worth the added safety check.

Reply:
Will do